### ISSUE TITLE
apoc.export.cypher.all exports unique indexes as node keys

### ISSUE THREAD

**Issue by thordal-indykite** [neo4j/apoc#345]
On Neo4J 4.4 using APOC 4.0

## Expected Behavior (Mandatory)

`apoc.export.cypher.all` should know the difference between Node Keys and Unique indexes

## Actual Behavior (Mandatory)

Given a unique constraint created with 
``` 
CREATE CONSTRAINT unique_Property_credentialUniqueScope_value IF NOT EXISTS FOR (n:Property)
REQUIRE (n.credential_unique_scope, n.value) IS UNIQUE;
``` 
it is exported as 
```
CREATE CONSTRAINT unique_Property_credentialUniqueScope_value ON (node:Property) ASSERT (node.credential_unique_scope, node.value) IS NODE KEY;
```

**Comment by Lojjs** [comment]
@thordal-indykite Thanks for reporting, this clearly looks like a bug. I will let you know once we have a fix.

**Comment by arxeiss** [comment]
When we were trying to reproduce that with @thordal-indykite we noticed that
- If UNIQUE is on a single Node attribute, it is exported correctly.
- If UNIQUE is set on pair of node attributes ( `credential_unique_scope` and `value` in this case) it is exported badly.

**Comment by Lojjs** [comment]
@thordal-indykite This will be fixed in the upcoming APOC versions 5.8.0 and 4.4.0.17
